src/views/Hearing/index.js

```diff
@@ -225,12 +226,14 @@ function groupSections(sections) {
     const sectionGroup = find(sectionGroups, group => section.type === group.type);
     if (sectionGroup) {
       sectionGroup.sections.push(section);
+      sectionGroup.n_comments += section.n_comments;
```

Comment:
It's still super duper unclean for a child component to mutate a prop.

Could this calculation be done in the `groupSections()` fn instead?